### ISSUE TITLE
Cannot login on fresh install of AsgardCMS

### ISSUE THREAD

**Issue by leonom** [AsgardCms/Platform#674]
### Summary of issue
Hello! Today after fresh install asgardcms via composer i try to login and see error:
"Undefined variable: alternate (View: /sites/personal/laravel/asgard/Themes/Flatly/views/layouts/master.blade.php) (View: /sites/personal/laravel/asgard/Themes/Flatly/views/layouts/master.blade.php)". 

### Steps to reproduce
composer create-project asgardcms/platform asgard
php artisan asgard:install

### System Details

- Operating System: CentOS 7.6.1810
- PHP Version: 7.3.2
- Laravel Version: 5.5.45
- `Asgardcms/Platform` version: 3.6.1

**Comment by jmazurpl** [comment]
I think is more or less the same issue as #676.
Delete the two letters indicating the language from the link:

http://your_address/**en**/backend

**Comment by Kiee** [comment]
@jmazurpl For me the problem was the reverse of your suggestion. I had to add /en/ for the error to subside.

**Comment by jmazurpl** [comment]
@Kiee Probably it depends on configuration. You can set it to hide the default language version from the URL and then its just like you wrote.

**Comment by djokodonev** [comment]
I ended up adding the /en/ before backend, that fixed the issue for me.

**Comment by michapixel** [comment]
Wow, this bug exists for more than a year. 
now this bug exists, if you install blank and then switch language ...

**Comment by imagina** [comment]
Is it failing with the latest version? 

Installing with:

composer create-project asgardcms/platform=4.0.x-dev your-project-name

?

**Comment by michapixel** [comment]
i installed it like it's described in the manual. resulting in Version Version 3.6.1

**Comment by imagina** [comment]
No, that's a very old version of the system @michapixel . Please install with:

composer create-project asgardcms/platform=4.0.x-dev your-project-name

@nWidart , I think you should tag 4.0x.
It's stable enough to have a final tag. What do you think?
